swc/private/swc.bzl

```diff
@@ -359,6 +364,22 @@ def _swc_impl(ctx):
                     output_sources.append(src)
                 continue
 
+            if _is_data_src(src_path):
```

Comment:
seems like this function name is begging for the json files to appear in a `data` attribute rather than `srcs`?
I don't remember if there's some reason that's not possible.

Reply:
I think it should be possible, but typescript accepts json as sources and part of ts compilation+type-checking. If rules_swc wants to provide a drop-in replacement for `ts_project(transpiler)` then I think we need to align with rules_ts treating json as sources...